storybook/src/components/plekinfo-card/plekinfo-card.core-template.ts

```diff
@@ -34,6 +37,17 @@ export const corePlekinfoCard: ComponentImplementation<PlekinfoCard<never>> = {
         ${symbool ? html`<span slot="symbol">${symbool}</span>` : nothing}
         ${html`<h2 slot="heading">${typeof label === "string" ? label : renvooiTemplate(label)}</h2>`}
         ${meta ? html`<div slot="meta">${labelTemplate(meta)}</div>` : nothing}
+        ${interactions && interactions.length > 0
```

Comment:
Wanneer `interactions` omgevorms is tot enkelvoudige `interaction` kan dit uitgekleed worden tot iets in de trant van

```ts
${interaction
          ? html`<div slot="interaction">
              <div class="dso-card-interaction">${slideToggleTemplate(interaction)}</div>
            </div>`
          : nothing}
```

Er zit ook nog wat styling gekoppeld aan `[slot="interactions"]` op r.145 dat we verliezen als we het slot `interaction` gaan noemen. Gaan we over bomen.